src/commonMain/kotlin/org/hisp/dhis/rules/models/RuleDataValueHistory.kt

```diff
@@ -1,6 +1,6 @@
 package org.hisp.dhis.rules.models
 
-import kotlinx.datetime.Instant
+import kotlin.time.Instant
 
 data class RuleDataValueHistory(
```

Comment:
This class is public but it is not directly used by the clients, is it? I don't mean we have to make it internal now (it could imply more changes), I just want to be sure that `Instant` is not exposed in the public API. @enricocolasante

Reply:
Yes, it is only used internally and yes, changing it to internal is not straightforward as more changes needs to be done